src/System.Net.NetworkInformation/tests/FunctionalTests/IPInterfacePropertiesTest_Windows.cs

```diff
@@ -21,6 +21,7 @@ public IPInterfacePropertiesTest_Windows()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "NetworkInterface.GetAllNetworkInterfaces not implemented for Uap. By design")]
```

Comment:
NetworkInformation work is tracked by #20014. 
We can't say right now if something is "by design" or not.

Please replace all SkipOnTargetFramework with ActiveIssue. We'll need a separate issue to track all tests that have been disabled for UWP (any flavor). Please assign the issue to me to ensure we include it in the correct workitem.